src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -38,12 +38,21 @@ Drawer {
   property var captureButtonIcon: QgsQuick.Utils.getThemeIcon("ic_camera_alt_border")
   property var confirmButtonIcon: QgsQuick.Utils.getThemeIcon("ic_check_black")
   property var cancelButtonIcon: QgsQuick.Utils.getThemeIcon("ic_clear_black")
+  property var backButtonSource: QgsQuick.Utils.getThemeIcon("ic_back")
```

Comment:
Let's move the getThemeIcon() calls to qgsquickfeatureformstyling.qml so it's easier to override them when needed...